src/reducers/pokemonReducer.js

```diff
@@ -0,0 +1,20 @@
+const INITIAL_STATE = {
+    list: [],
+    pokemonSelected: null,
```

Comment:
`pokemonSelected` also this is not needed in Redux because you can read the ID each time from the URL param on detail page.

Reply:
Yes, it's just I was trying to use more Redux even if the project was small, I just felt like you could find more use of redux in the code.